src/pages/Faq/Faq.js

```diff
@@ -1,7 +1,180 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import variables from "styles/variables";
+import ballon from "asset/images/talk-ballon.png";
+import FaqModal from "./FaqModal";
 
 const Faq = () => {
-  return <div>여기는 자주하는 질문</div>;
+  const [faqData, setFaqData] = useState([{}]);
```

Comment:
```suggestion
  const [faqData, setFaqData] = useState([]);
```
목데이터 초기값을 보니 배열로 감싸진 객체들이네요!
초기값은 위와같이 구성하는 것이 좋겠습니다.